Index: LanceOfDestiny-FourthDimension/src/main/java/org/LanceOfDestiny/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/LanceOfDestiny-FourthDimension/src/main/java/org/LanceOfDestiny/Main.java b/LanceOfDestiny-FourthDimension/src/main/java/org/LanceOfDestiny/Main.java
--- a/LanceOfDestiny-FourthDimension/src/main/java/org/LanceOfDestiny/Main.java	
+++ b/LanceOfDestiny-FourthDimension/src/main/java/org/LanceOfDestiny/Main.java	
@@ -1,8 +1,12 @@
-package org.LanceOfDestiny;
+package main.java.org.LanceOfDestiny;
+
+import main.java.org.LanceOfDestiny.UI.WindowCreater;
 
 public class Main {
     public static void main(String[] args) {
         System.out.println("Helloodssssdsddsddsrld!");
 
+        WindowCreater wC = WindowCreater.getInstance();
+
     }
 }
Index: LanceOfDestiny-FourthDimension/src/test/java/org/LanceOfDestiny/ExampleTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/LanceOfDestiny-FourthDimension/src/test/java/org/LanceOfDestiny/ExampleTest.java b/LanceOfDestiny-FourthDimension/src/test/java/org/LanceOfDestiny/ExampleTest.java
--- a/LanceOfDestiny-FourthDimension/src/test/java/org/LanceOfDestiny/ExampleTest.java	
+++ b/LanceOfDestiny-FourthDimension/src/test/java/org/LanceOfDestiny/ExampleTest.java	
@@ -1,4 +1,4 @@
-package org.LanceOfDestiny;
+package test.java.org.LanceOfDestiny;
 
 import org.junit.jupiter.api.Test;
 
Index: LanceOfDestiny-FourthDimension/src/main/java/org/LanceOfDestiny/UI/WindowCreater.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/LanceOfDestiny-FourthDimension/src/main/java/org/LanceOfDestiny/UI/WindowCreater.java b/LanceOfDestiny-FourthDimension/src/main/java/org/LanceOfDestiny/UI/WindowCreater.java
new file mode 100644
--- /dev/null	
+++ b/LanceOfDestiny-FourthDimension/src/main/java/org/LanceOfDestiny/UI/WindowCreater.java	
@@ -0,0 +1,25 @@
+package main.java.org.LanceOfDestiny.UI;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class WindowCreater {
+
+    private JFrame frame;
+    private static WindowCreater instance;
+
+    private WindowCreater(){
+
+    }
+
+    public static WindowCreater getInstance(){
+        if(instance == null){
+            instance = new WindowCreater();
+        }
+        return instance;
+    }
+
+    public JFrame getFrame() {
+        return frame;
+    }
+}
Index: LanceOfDestiny-FourthDimension/src/main/java/org/LanceOfDestiny/UI/Window.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/LanceOfDestiny-FourthDimension/src/main/java/org/LanceOfDestiny/UI/Window.java b/LanceOfDestiny-FourthDimension/src/main/java/org/LanceOfDestiny/UI/Window.java
new file mode 100644
--- /dev/null	
+++ b/LanceOfDestiny-FourthDimension/src/main/java/org/LanceOfDestiny/UI/Window.java	
@@ -0,0 +1,17 @@
+package main.java.org.LanceOfDestiny.UI;
+
+import javax.swing.*;
+
+public class Window {
+
+    private JFrame frame;
+
+
+    public Window(String name){
+        frame = new JFrame(name);
+
+    }
+
+
+
+}
Index: .idea/LanceOfDestiny-FourthDimension.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/LanceOfDestiny-FourthDimension.iml b/.idea/LanceOfDestiny-FourthDimension.iml
--- a/.idea/LanceOfDestiny-FourthDimension.iml	
+++ b/.idea/LanceOfDestiny-FourthDimension.iml	
@@ -2,8 +2,36 @@
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="true">
     <exclude-output />
-    <content url="file://$MODULE_DIR$" />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/LanceOfDestiny-FourthDimension/src" isTestSource="false" />
+    </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library">
+      <library name="JUnit4">
+        <CLASSES>
+          <root url="jar://$MAVEN_REPOSITORY$/junit/junit/4.13.1/junit-4.13.1.jar!/" />
+          <root url="jar://$MAVEN_REPOSITORY$/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar!/" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+      </library>
+    </orderEntry>
+    <orderEntry type="module-library">
+      <library name="JUnit5.8.1">
+        <CLASSES>
+          <root url="jar://$MAVEN_REPOSITORY$/org/junit/jupiter/junit-jupiter/5.8.1/junit-jupiter-5.8.1.jar!/" />
+          <root url="jar://$MAVEN_REPOSITORY$/org/junit/jupiter/junit-jupiter-api/5.8.1/junit-jupiter-api-5.8.1.jar!/" />
+          <root url="jar://$MAVEN_REPOSITORY$/org/opentest4j/opentest4j/1.2.0/opentest4j-1.2.0.jar!/" />
+          <root url="jar://$MAVEN_REPOSITORY$/org/junit/platform/junit-platform-commons/1.8.1/junit-platform-commons-1.8.1.jar!/" />
+          <root url="jar://$MAVEN_REPOSITORY$/org/apiguardian/apiguardian-api/1.1.2/apiguardian-api-1.1.2.jar!/" />
+          <root url="jar://$MAVEN_REPOSITORY$/org/junit/jupiter/junit-jupiter-params/5.8.1/junit-jupiter-params-5.8.1.jar!/" />
+          <root url="jar://$MAVEN_REPOSITORY$/org/junit/jupiter/junit-jupiter-engine/5.8.1/junit-jupiter-engine-5.8.1.jar!/" />
+          <root url="jar://$MAVEN_REPOSITORY$/org/junit/platform/junit-platform-engine/1.8.1/junit-platform-engine-1.8.1.jar!/" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+      </library>
+    </orderEntry>
   </component>
 </module>
\ No newline at end of file
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	
+++ b/.idea/misc.xml	
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager">
+  <component name="ProjectRootManager" version="2" project-jdk-name="17" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
